### PR TITLE
Feature/#171 contactmoment logging

### PR DESCRIPTION
**Pull request by lencodes**
~~WIP: the collected data still needs to be sent to the gateway, the rest can be reviewed.~~

Done.

### CODE REVIEW

File: src/features/werkbericht/WerkBericht.vue [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
     </small>
 
     <div class="heading-container">
+      <div class="heading-top-row">
+        <time :datetime="bericht.date.toISOString()">{{
+          localeString(bericht.date)
+        }}</time>
+
+        <menu>
+          <input
```
Comment (felixcicatt) [inline src/features/werkbericht/WerkBericht.vue:14]: I had some doubts about screen reader support for an input with only a title attribute, [but it actually seems widely supported](https://www.powermapper.com/tests/screen-readers/labelling/input-text-title/). Good to know!

File: src/features/werkbericht/WerkBericht.vue [hunk 2]
```diff
@@ -66,6 +80,22 @@ const props = defineProps({
   },
 });
 
+const contactmomentStore = useContactmomentStore();
+
+const berichtSelectedInContactmoment = computed(() => {
+  const foundInNieuwsberichten =
+    contactmomentStore.nieuwsberichten.findIndex(
```
Comment (felixcicatt) [inline src/features/werkbericht/WerkBericht.vue:87]: I prefer the `some` method, but that's a matter of preference I guess.

**Review by felixcicatt — CHANGES_REQUESTED** [review]
Good work! I'll review the actual saving once you're done with it.

File: src/features/search/GlobalSearch.vue [hunk 3]
```diff
@@ -54,9 +54,8 @@
                 </a>
                 <a
                   v-else
-                  :href="url.toString()"
```
[the remark this answers is not in the document]
Comment (lencodes) [reply src/features/search/GlobalSearch.vue:57]: Fixed.

File: src/features/werkbericht/WerkBericht.vue [hunk 4]
```diff
@@ -1,26 +1,39 @@
...
       {{ bericht.types.join(", ") }}
     </small>
 
     <div class="heading-container">
+      <div class="heading-top-row">
+        <time :datetime="bericht.date.toISOString()">{{
+          localeString(bericht.date)
+        }}</time>
+
+        <menu>
```
[the remark this answers is not in the document]
Comment (lencodes) [reply src/features/werkbericht/WerkBericht.vue:13]: Never really thought about this, I don't usually use `menu` tags, however I did see them being used within this project. Did some research, came to the following conclusions:

1. The `menu` tag is [barely supported by modern browsers](https://www.w3schools.com/tags/tag_menu.asp#:~:text=controls%20and%20commands.-,Browser%20Support,table%20specify%20the%20first%20browser%20version%20that%20fully%20supports%20the%20element.,-Element), so I think using it altogether might be a bad idea;
2. I then though of using the `nav` tag, however [it's not intended to be used outside of main navigations](https://www.w3schools.com/tags/tag_nav.asp#:~:text=Notice%20that%20NOT%20all%20links%20of%20a%20document%20should%20be%20inside%20a%20%3Cnav%3E%20element.%20The%20%3Cnav%3E%20element%20is%20intended%20only%20for%20major%20blocks%20of%20navigation%20links.) (which this one is not, in my opinion)
3. I now opted to just put it in a `div`, as I didn't think that it's a list with items that are relevant to each other.

Thoughts? @felixcicatt

[the remark this answers is not in the document]
Comment (felixcicatt) [reply src/features/werkbericht/WerkBericht.vue]: O wow, I was confused for a moment but apparently w3schools is plain wrong! The table they link to is about the `<menuitem>` tag, even . Indeed, [that tag is very poorly supported](https://caniuse.com/menu). Fortunately, the `<menu>` tag itself [is widely supported](https://caniuse.com/mdn-html_elements_menu). I've seen quite a number of mistakes on w3schools by now which has made me distrust it as a knowledge base. Mdn web docs tends to be more accurate. [This is they're entry about the `<menu>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu).

**Review by felixcicatt — APPROVED** [review]

**Pull request merged**